tests/test_creator.cairo

```diff
@@ -0,0 +1,590 @@
+
+    # check the game is properly stored
+    let (game_len, game) = view_game(game_index=game_index, generation=generation)
+    assert game_len = 225
+    assert game[0] = 0
+    assert game[1] = 0
+    assert game[2] = 0
+    assert game[3] = 0
+    assert game[4] = 0
+    assert game[5] = 0
+    assert game[6] = 0
+    assert game[7] = 0
+    assert game[8] = 0
+    assert game[9] = 0
+    assert game[10] = 0
+    assert game[11] = 0
+    assert game[12] = 0
+    assert game[13] = 0
+    assert game[14] = 0
+    assert game[15] = 0
+    assert game[16] = 0
+    assert game[17] = 0
+    assert game[18] = 0
+    assert game[19] = 0
+    assert game[20] = 0
+    assert game[21] = 0
+    assert game[22] = 0
+    assert game[23] = 0
+    assert game[24] = 0
+    assert game[25] = 0
+    assert game[26] = 0
+    assert game[27] = 0
+    assert game[28] = 0
+    assert game[29] = 0
+    assert game[30] = 0
+    assert game[31] = 0
+    assert game[32] = 0
+    assert game[33] = 0
+    assert game[34] = 0
+    assert game[35] = 0
+    assert game[36] = 0
+    assert game[37] = 0
+    assert game[38] = 0
+    assert game[39] = 0
+    assert game[40] = 0
+    assert game[41] = 0
+    assert game[42] = 0
+    assert game[43] = 0
+    assert game[44] = 0
+    assert game[45] = 0
+    assert game[46] = 0
+    assert game[47] = 0
+    assert game[48] = 0
+    assert game[49] = 0
+    assert game[50] = 0
+    assert game[51] = 0
+    assert game[52] = 0
+    assert game[53] = 0
+    assert game[54] = 0
+    assert game[55] = 0
+    assert game[56] = 0
+    assert game[57] = 0
+    assert game[58] = 0
+    assert game[59] = 0
+    assert game[60] = 0
+    assert game[61] = 0
+    assert game[62] = 0
+    assert game[63] = 0
+    assert game[64] = 0
+    assert game[65] = 0
+    assert game[66] = 0
+    assert game[67] = 0
+    assert game[68] = 0
+    assert game[69] = 0
+    assert game[70] = 0
+    assert game[71] = 0
+    assert game[72] = 0
+    assert game[73] = 0
+    assert game[74] = 0
+    assert game[75] = 0
+    assert game[76] = 0
+    assert game[77] = 0
+    assert game[78] = 0
+    assert game[79] = 0
+    assert game[80] = 0
+    assert game[81] = 0
+    assert game[82] = 0
+    assert game[83] = 0
+    assert game[84] = 0
+    assert game[85] = 0
+    assert game[86] = 0
+    assert game[87] = 0
+    assert game[88] = 0
+    assert game[89] = 0
+    assert game[90] = 0
+    assert game[91] = 0
+    assert game[92] = 0
+    assert game[93] = 0
+    assert game[94] = 0
+    assert game[95] = 0
+    assert game[96] = 0
+    assert game[97] = 0
+    assert game[98] = 0
+    assert game[99] = 0
+    assert game[100] = 0
+    assert game[101] = 0
+    assert game[102] = 0
+    assert game[103] = 0
+    assert game[104] = 0
+    assert game[105] = 0
+    assert game[106] = 0
+    assert game[107] = 0
+    assert game[108] = 0
+    assert game[109] = 0
+    assert game[110] = 0
+    assert game[111] = 0
+    assert game[112] = 1
```

Comment:
I think it would be easier to understand if `1` created a board with the last cell alive. This way binary representation directly translated to index.

Reply:
Yeah, I was thinking about it too, but the problem is the board is actually split into half (high and low values) to store in one felt. 

So the one way we could achieve this I can see, is "changing" what high and low values are, that would also mean we should probably change the arrays lenghts. 

That would work for first 113 indices, but still wouldn't for the next 112, that's why I just left this as it is, but perhaps it is worth it? thoughts, or maybe some ideas other than that?

Reply:
Yes, the dimensions used would be changed: we would use 128 bits from lower and only 97 bits from higher one. I think we should change it. This way the binary representation translates directly to the board:
```
ZEROES_PREFIX + 10000000000000000000000...0000000000000000000000000000001
                ^ first cell (0,0), followed by cells in first row      ^last cell (14,14)
```

or we can do it in reverse if this requires less computations.